client/src/Main.elm

```diff
@@ -884,23 +987,23 @@ pieLabel slice ( label, _ ) =
         [ TypedSvg.Core.text label ]
```

Comment:
I don't think you really intended to use SVG here?

Reply:
I use SVG for the visual representation of the construct.